### ISSUE TITLE
Command 'eslint...' not found since updating to 2.4.4

### ISSUE THREAD

**Issue by the21st** [microsoft/vscode-eslint#1777]
Hi there, since I upgraded to 2.4.4, I am getting this error when I try to invoke any eslint command from the command palette:

![image](https://github.com/microsoft/vscode-eslint/assets/4610874/cc0ab560-f683-4a53-84b9-50f56c93245b)

Downgrading the plugin to 2.4.2 seems to resolve the problem.

**Comment by RussOakham-BG** [comment]
Had the same when trying to restart my ESLint server locally, as linting was no longer showing in-document on VSCode (No red squiggles).

Would get `command eslint.restart not found` error upon trying.

Rolling back to the prior version resolved it for me in the short term.

**Comment by dbaeumer** [comment]
Anything in the ESLint output channel ?

**Comment by 6feferonka9** [comment]
I second this, latest version is broken. No command works.

Only output is: The language client requires VS Code version ^1.82.0 but received version 1.80.0

After updating VS code it seems to be working

**Comment by yisibell** [comment]
I got Same problem in 2.4.4, so that i have to use 2.4.2. And there is nothing in output channel.

**Comment by yisibell** [comment]
@dbaeumer 

![ff9707557768478d5424300b68833f6](https://github.com/microsoft/vscode-eslint/assets/20988580/2a59de20-f96f-4ed6-898d-97bc85b01f71)

**Comment by the21st** [comment]
> @yisibell which version of VS Code are you using?

I was on vscode 1.81.1. After updating to 1.85.2, the issue seems to be resolved 🎉

**Comment by dbaeumer** [comment]
Yes, version 2.4.4 of the extension requires at least 1.82.0 of VS Code. I made a change to the Extension that will avoid that the extension gets installed in the future when the minimum VS Code version is not available. I the meantime all I can recommend to either upgrade to 1.82.0 of VS Code or to downgrade the extension to 2.4.2.

**Comment by michael-chan-mox** [comment]
> Yes, version 2.4.4 of the extension requires at least 1.82.0 of VS Code. I made a change to the Extension that will avoid that the extension gets installed in the future when the minimum VS Code version is not available. I the meantime all I can recommend to either upgrade to 1.82.0 of VS Code or to downgrade the extension to 2.4.2.

Same here, upgrading VsCode works!

**Comment by Germxu** [comment]
I found the same issue in version 3.0.8, and I revert to version 2.4.4.
![image](https://github.com/microsoft/vscode-eslint/assets/20300863/abc00113-6398-41ad-ae3c-2262ae690900)

**Comment by dbaeumer** [comment]
@Germxu which VS Code version are you using?

**Comment by Germxu** [comment]
> @Germxu which VS Code version are you using?
I don't know why it didn’t check for updates.

![image](https://github.com/microsoft/vscode-eslint/assets/20300863/7324b670-205b-4795-8f20-07d0b6ef83f2)

**Comment by dbaeumer** [comment]
@Germxu can you update to the latest 1.90.x version. This should make it work again.

**Comment by PRO-2684** [comment]
Ran into the same issue. Using VSCode 1.95.3, ESLint plugin v3.0.10. Tried switching to PowerShell as default profile, but no luck either. (Seems that this plugin does not respect `terminal.integrated.automationProfile`)

![Image](https://github.com/user-attachments/assets/42b6103d-c86e-4be7-bb04-c630dc65ffc0)

**Comment by dbaeumer** [comment]
@PRO-2684 the eslint checking done by this plugin is not executet in a terminal. I  guess you see another problem. Can you please open a new issue with steps on how to reproduce. Best would be a GitHub repository to clone.

**Comment by dbaeumer** [comment]
This issue has been closed because it needs more information and has not had recent activity. See also our [issue reporting](https://aka.ms/vscodeissuereporting) guidelines.

Happy Coding!
